modules/programs/rtx.nix

```diff
@@ -0,0 +1,75 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.programs.rtx;
+  tomlFormat = pkgs.formats.toml { };
+in {
+  meta.maintainers = [ hm.maintainers.pedorich-n ];
+
+  options = {
+    programs.rtx = {
+      enable = mkEnableOption "RTX. Runtime Executor (asdf Rust clone)";
+
+      package = mkPackageOption pkgs "rtx" { };
+
+      enableBashIntegration = mkEnableOption "Bash Integration";
+
+      enableZshIntegration = mkEnableOption "Zsh Integration";
+
+      enableFishIntegration = mkEnableOption "Fish Integration";
```

Comment:
Should these default to `true`?

Reply:
I guess. 
I checked, and it looks like this value is set to `true` for most of the programs by default.